app/extensions/brave/content/scripts/pageInformation.js

```diff
@@ -156,7 +156,7 @@
       try {
         if (!resolve(rule.condition, context)) continue
       } catch (ex) {
-        console.log('error resolving rule at position #' + i + '\n' + ex.stack)
+        console.error('error resolving rule at position #' + i + '\n' + ex.stack)
```

Comment:
Could you change this to just log the exception instead? (just `ex`, instead of `ex.stack`).  console.error will already log a stack trace 😄